distribution/docs/sender/application.properties

```diff
@@ -232,3 +229,24 @@ pgp.sender.password=
 # User name of the PGP receiver key pair
 pgp.sender.receiverUserId=
 # ----------------------------------------------------------------------------------------------------------------------
+
+# ***************************** Application Status Notification: Emails Configuration ******************************************#
+#The list of email addresses to be notified when the application stops. Multiples addresses must be separated by comma
+spring.email.send-to=
+#Path to application log file
+spring.email.attachment.application-log-file=${eip.home}${file.separator}shared${file.separator}logs${file.separator}eip${file.separator}openmrs-eip.log
```

Comment:
This is not the default log file location, it is **${eip.home}/logs/openmrs-eip.log**